datasets/karate-disjoint.csv

```diff
@@ -0,0 +1,312 @@
+1 0 1.0
```

Comment:
Since karate dataset has no edge direction, it would be better if the file stored undirected edges. So that cugraph would then reconstruct the symmetric matrix. 
This would apply to previous csv datasets containing undirected graphs too.